pkg/apis/litmuschaos/v1alpha1/chaosengine_types.go

```diff
@@ -27,8 +27,8 @@ import (
 type ChaosEngineSpec struct {
 	//Appinfo contains deployment details of AUT
 	Appinfo ApplicationParams `json:"appinfo"`
-	//ChaosType define wheather it is an infra chaos or app chaos
-	ChaosType string `json:"chaosType,omitempty"`
+	//AnnotationCheck define wheather annotation is true or false
```

Comment:
Please provide more informative comment like :
AnnotationCheck defines whether annotation check is allowed or not. It can be true or false